index.php

```diff
@@ -0,0 +1,40 @@
+<?php
+include("Include/init.php"); 
+$stories = getAllStories();
+
+echo "
+    <!DOCTYPE html
+    <head>
+        <meta charset='utf-8'>
+
+        <style>
+            .grid-container {
```

Comment:
I think this file is completely going to change by the time you're finished with the design so I'm not going to request any changes here